tests/behat/serve-bootstrap.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+use SilverStripe\CMS\Model\SiteTree;
```

Comment:
`use` statements are safe to use if the class doesn't exist; SiteTree::class itself doesn't trigger autoloading, so do we feel concerned about this being here?

i.e. `class_exists(SiteTree::class)` will safely return false without the CMS module.